OWNERS

```diff
@@ -0,0 +1,24 @@
+# Reviewers can /lgtm /approve but not sufficient for auto-merge without an
+# approver
+reviewers:
+- Rajakavitha1
```

Comment:
Can we not use Github groups?

Reply:
>We use aliases for groups instead of GitHub Teams, because changes to GitHub Teams are not publicly auditable.

https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#owners_aliases

That was true the time these files were initially written, github's teams have improved and that may no longer be true, we're planning to revisit that down the line